src/fides/api/service/connectors/scylla_query_config.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+from fides.api.graph.config import Field
+from fides.api.models.policy import Policy
+from fides.api.service.connectors.query_config import SQLLikeQueryConfig
+
+ScyllaDBStatement = Tuple[str, Dict[str, Any]]
+"""
+A ScyllaDB statement is a tuple whose first component is the query string
+and whose second component is a dict of parameters to pass in to the query.
+
+E.g
+('SELECT name FROM users WHERE user.id = %(user_id)s', { "user_id": 123 })
+"""
+
+
+class ScyllaDBQueryConfig(SQLLikeQueryConfig[ScyllaDBStatement]):
+    def dry_run_query(self) -> Optional[str]:
+        """dry run query for display"""
+        return None
+
+    def query_to_str(
+        self, t: ScyllaDBStatement, input_data: Dict[str, List[Any]]
+    ) -> Optional[str]:
+        """Convert query to string"""
+        return None
```

Comment:
I wasn't sure if I needed to implement these, I saw that some connectors implement them and some don't , not sure how to tell if it's needed